lib/thirdparty/LBRY.class.php

```diff
@@ -5,6 +5,10 @@ class LBRY
 {
     public static function getApiUrl($endpoint)
     {
+        if (!strlen(Config::get(Config::LBRY_API_SERVER)) > 0) {
+            throw new Exception("API server URL is missing from configuration");
```

Comment:
@NetOperatorWibby This might seem small, but even referencing the config path creates a difficult to trace dependency (if someone was refactoring configuration logic, they shouldn't be expected to update this unrelated file if the API signature of `Config` didn't change).